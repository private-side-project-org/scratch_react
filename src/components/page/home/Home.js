import React from 'react';
// import Navbar from './Navbar';

const Home = () => {
  return (
    <div className="page-container home-container">
      <span className="home-title">Feel The Nature</span>
    </div>
  );
};

export default Home;
