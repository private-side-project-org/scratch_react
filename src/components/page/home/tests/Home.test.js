import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../Home';

it('should render', () => {
  const wrapper = renderer.create(<Home />);
  expect(wrapper).toMatchSnapshot();
});
