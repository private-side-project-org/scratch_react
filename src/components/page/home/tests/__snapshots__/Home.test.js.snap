// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<div
  className="page-container home-container"
>
  <span
    className="home-title"
  >
    Feel The Nature
  </span>
</div>
`;
